packages/react-aria-components/src/Tree.tsx

```diff
@@ -128,7 +130,15 @@ export interface TreeProps<T> extends Omit<AriaTreeProps<T>, 'children'>, Multip
    * Whether `disabledKeys` applies to all interactions, or only selection.
    * @default 'all'
    */
-  disabledBehavior?: DisabledBehavior
+  disabledBehavior?: DisabledBehavior,
+  /** The drag and drop hooks returned by `useDragAndDrop` used to enable drag and drop behavior for the Tree. */
+  dragAndDropHooks?: DragAndDropHooks,
+
+  /**
+   * An optional keyboard delegate implementation for type to select,
+   * to override the default.
+   */
+  keyboardDelegate?: KeyboardDelegate
```

Comment:
Do we need to expose this as a public prop? Doesn't look like we do in other collection components e.g. GridList